src/main/java/Circle.java

```diff
@@ -0,0 +1,26 @@
+public class Circle extends Figure {
+    private long radius;
+    private String color;
+
+    public Circle(long radius, String color) {
+        this.radius = radius;
+        this.color = color;
+    }
+
+    public String color() {
+        return color;
+    }
+
+    public double area() {
```

Comment:
this method inherited from Figure. Don't forget to write @Override https://www.tutorialspoint.com/importance-of-override-annotation-in-java